jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/SimpleRuntimeEnvironment.java

```diff
@@ -106,8 +109,17 @@ public void init() {
      * @param type type of the asset
      */
     public void addAsset(Resource resource, ResourceType type) {
-        this.kbuilder.add(resource, type);
-        if (this.kbuilder.hasErrors()) {            
+        if (resource.getSourcePath() != null && resource.getSourcePath().endsWith(".csv")) {
+            Properties prop = new Properties();
+            prop.setProperty(ResourceTypeImpl.KIE_RESOURCE_CONF_CLASS, DecisionTableConfigurationImpl.class.getName());
+            prop.setProperty(DecisionTableConfigurationImpl.DROOLS_DT_TYPE, DecisionTableInputType.CSV.toString());
+            ResourceConfiguration conf = ResourceTypeImpl.fromProperties(prop);
+            this.kbuilder.add(resource, type, conf);
```

Comment:
can't we handle that in kbuilder itself? Reason I ask is that other users might run into exactly same issue when not using RuntimeManager API